packages/livechat/src/widget.ts

```diff
@@ -225,6 +226,15 @@ const openWidget = () => {
 	emitCallback('chat-maximized');
 };
 
+const setWidgetPosition = (position: 'left' | 'right' = 'right') => {
+	if (!widget) {
```

Comment:
Let's give a warn if the position is invalid, so the user knows that it did not work